gui/src/app/components/FileEditor/TextEditor.tsx

```diff
@@ -97,24 +104,34 @@ const TextEditor: FunctionComponent<Props> = ({
   }, [codeMarkers, monacoInstance, editorInstance]);
 
   useEffect(() => {
-    if (!editorInstance) return;
+    if (!editorInstance || !monacoInstance) return;
     if (!contentOnEmpty) return;
     if (text || editedText) {
       return;
     }
-    const contentWidget = createHintTextContentWidget(contentOnEmpty);
+    const widgetPosition = {
+      position: { lineNumber: 1, column: 1 },
+      preference: [monacoInstance.editor.ContentWidgetPositionPreference.EXACT],
+    };
```

Comment:
Seems like this ought to be `const`-able, but the linter doesn't like it. Probably doesn't matter anyway...

Reply:
Not sure I follow what you mean by const-able in this context?

Reply:
I mean annotated as `as const`, since its members don't seem to be able to be changed anywhere in the code. But that conflicts with the looser typing further down.

(The larger context was me wondering if it should be memoized, before realizing this was already in a `useEffect` hook, so it doesn't matter.)